tests/test_run_metrics.py

```diff
@@ -41,24 +37,25 @@ def test_run_metrics():
     mapping = [(n, n) for n in graph.nodes()]
 
     # Check matcher input -- not instantiated
-    with pytest.raises(TypeError):
-        run_metrics(graph, graph, DummyMatcher, [DummyMetric()])
+    # with pytest.raises(TypeError):
+    #     run_metrics(graph, graph, DummyMatcher, [DummyMetric()])
 
-    # Check matcher input -- wrong type
-    with pytest.raises(TypeError):
-        run_metrics(graph, graph, "rando", DummyMetric())
+    # # Check matcher input -- wrong type
+    # with pytest.raises(TypeError):
+    #     run_metrics(graph, graph, "rando", DummyMetric())
 
-    # Check metric input -- not instantiated
-    with pytest.raises(TypeError):
-        run_metrics(graph, graph, DummyMatcher(), [DummyMetric])
+    # # Check metric input -- not instantiated
+    # with pytest.raises(TypeError):
+    #     run_metrics(graph, graph, DummyMatcher(), [DummyMetric])
 
-    # Check metric input -- wrong type
-    with pytest.raises(TypeError):
-        run_metrics(graph, graph, DummyMatcher(), [DummyMetric(), "rando"])
+    # # Check metric input -- wrong type
+    # with pytest.raises(TypeError):
```

Comment:
Curious why we've commented these out, was that just testing purposes?

Reply:
Yup my bad. Just from debugging something